pkg/consts/defaults.go

```diff
@@ -23,3 +23,9 @@ const DefaultName = "default"
 const DefaultMedium = "default"
 
 const MaxSlotLocationReserve = 10 << 30 // 10GiB
+
+const DefaultStrawberryControllerFamily = "chyt"
+
+func GetDefaultStrawberryControllerFamilies() []string {
+	return []string{"chyt", "jupyt"}
+}
```

Comment:
I don't understand, should the default family be `chyt` or list of of `chyt, jupyt`?
Maybe first one better be called just `DefaultStrawberryFamily`, not ControllerFamily according to static config values?

Reply:
Poor naming :(
I renamed DefaultStrawberryControllerFamily to DefaultRouteStrawberryControllerFamily - that's explains better what it is about (and it is about the family that will receive requests for domains that are not explicitly specified in `http_controller_mappings`)